libs/labelbox/tests/integration/test_user_management.py

```diff
@@ -1,10 +1,13 @@
-from labelbox import ProjectRole
 import pytest
 
+from labelbox import ProjectRole
+from faker import Faker
+
+faker = Faker()
 
 def test_org_invite(client, organization, environ, queries):
     role = client.get_roles()['LABELER']
-    dummy_email = "none@labelbox.com"
+    dummy_email = faker.email()
```

Comment:
I just hope this doesn't generate email accounts that exist.

Reply:
As in, I don't want emails to go out to users.

Reply:
Good point, let me use a random string at labelbox